IOMC/Input/src/MCFileSource.cc

```diff
@@ -32,7 +33,8 @@ MCFileSource::MCFileSource(const ParameterSet & pset, InputSourceDescription con
   }  
   
   reader_->initialize(fileName);  
-  produces<HepMCProduct>();
+  produces<HepMCProduct>("generator");
```

Comment:
Sources are not allowed to have a label. However, Source are allowed to provide data products from 'earlier' Process. Therefore it would be possible to have the source create a data product using the 'generator' module label for a made up earlier process (say 'MCFileGen'). However the base class used by `MCFileSource` does not support creation of 'earlier' Processes.